### ISSUE TITLE
How to get started with new Languages

### ISSUE THREAD

**Issue by thebergamo** [dwyl/start-here#100]
I like start learn new languages with a project based to do this. The idea behind it is create a repository with ideas to create projects with different levels start with simpler and growing to more complex projects to try and test new languages;

What you guys/ladies thinking about this?

**Comment by nelsonic** [comment]
@thebergamo this is _exactly_ how we learn. hence all the [`learn-xyz`](https://github.com/search?q=org%3Adwyl+learn) repos in `dwyl` 😉 

The _reason_ we haven't created any new _language_-specific repos e.g `learn-elm`, `learn-go`, `learn-erlang`, `learn-elixir`, `learn-python`, `learn-julia` or `learn-rust` is because we don't think we have [_mastered_](https://en.wikipedia.org/wiki/Mastery_learning) our _existing_ Tech Stack: https://github.com/dwyl/technology-stack ... ⌛️ 

There is a _school_ of thought that _suggests_ learning a "_new_" language will help you _think_ in a new way an see your _existing_ code with _new_ perspective ... e.g. learning `clojure` to learn about Functional Programming will help you with your JS FP ... 🤔 

While I _agree_ with this and have enjoyed learning _several_ programming languages over the years, I still think _most_ people have not _mastered_ Core JavaScript (Good Parts) https://github.com/dwyl/goodparts and to add a _whole other_ language/runtime in the mix would _only_ confuse them. 😖 

However, for the people who feel they have _mastered_ JS, we _should_ have a list of languages they should _consider_ learning _next_ ... what _language_ would you suggest and ***why***? ❓

**Comment by thebergamo** [comment]
@nelsonic I agree with your point, the idea is not creating a lot of repositories about any new or old languages. Is just give ideas about some projects to how to learn a new language.

For example, with languages, most common used in backend, like Go, Erlang, Rust and many others, create a simple path like: "Learn-backend" or just create an single repo with the ideas and micro tutorials about these themes.

The idea, is giving ideas to start learn something, but not specifying that thing.

```markdown
## Backend
[ ] Create a K/V database
[ ] Create  REST API with Auth and respond with JSON, JSONP and XML
[ ] Create a simple HTTP framework
[ ] ...

## Frontend
[ ] Create a TODO List
[ ] Create a Template Engine
[ ] ...

## Mobile
[ ] Create a CRUD for User management
[ ] Create a Client with an Open API
[ ] ...

```

This make sense?

**Comment by nelsonic** [comment]
Ahh... I _get_ it. sorry for being a bit _slow_ ... 🐌 
That sir is a _really_ good idea. 💡 
(_we have thought of having a suggested project list for http://www.foundersandcoders.com/programme/ but not explicitly listed them so far ... we are still writing "tutorials" for all the "basics" of our stack...!_)

**Comment by thebergamo** [comment]
I'm not sure, if I got the entire idea behind founders & coders.
But you guys, need to become this initiatives in South America too! Just btw, the idea is create projects, well descriptions to all this paths :D how we can proceed?
